integration-tests/ra/src/test/java/io/quarkus/it/artemis/ra/DisableDataBaseService.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.it.artemis.ra;
```

Comment:
Can we move this class in a separata package, e.g. `io.quarkus.it.artemis.ra.profile`?